tools/mkpkg/mkpkg_kodi

```diff
@@ -31,7 +31,7 @@ DEST_DIR="$PKG_NAME-$BRANCH"
 
 echo "getting sources..."
   if [ ! -d $DEST_DIR.git ]; then
-    git clone --depth 1 $GIT_REPO -b $BRANCH $DEST_DIR.git
+    git clone --depth 128 $GIT_REPO -b $BRANCH $DEST_DIR.git
```

Comment:
It's in the first comment.

By default `mkpkg_kodi master` will use the last commit (at the time, Beta1) when we actually wanted the commit before (Alpha3) in order to create the kodi-17-alpha3 package. The current `mkpkg_kodi` does take a second optional argument which is a specific rev, but with `--depth 1` it's not possible to select a different revision other than the last commit (Beta1).